spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/config/StandardizationConfigParser.scala

```diff
@@ -137,64 +148,66 @@ object StandardizationConfigParser {
         .action((value, config) => config.withFixedWidthTreatEmptyValuesAsNulls(Some(value)))
         .text("For FixedWidth file format. Treats empty values as null. Default is false"),
 
-      opt[String]("null-value").optional()
-        .action((value, config) => config.withFixedWidthNullValue(Some(value)))
-        .text("""For FixedWidth file format. Sets the representation of a null value. Defaults is ""."""),
-
       checkConfig(checkConfigX(_, builder))
     )
   }
   //scalastyle:on method.length
 
   private val formatsSupportingCharset = List("xml", "csv", "json", "cobol", "fixed-width")
 
-  private def typicalError(field: String, format: String): String = {
-    s"The $field option is supported only for $format format"
+  private def unsupportedOptionError(option: String, format: String): String = {
+    unsupportedOptionError(option, Seq(format))
+  }
+
+  private def unsupportedOptionError(option: String, formats: Seq[String]): String = {
+    def sprintf(frmt: String, s: String = ""): String = s"The $option option is supported only for $frmt format$s"
+
+    formats match {
+      case Seq()       => ""
+      case Seq(format) => sprintf(format)
+      case _           =>
+        val revertedFormats = formats.reverse
+        val frmt = revertedFormats.tail.reverse.mkString(", ") + " and " + revertedFormats.head
```

Comment:
I see your point here, but it seems a bit too much. I'd either fall back to good old techy `formats.mkString(", ")` or extract the logic to `mkStringWithAFancyEnumerationThatWillLookReallyCoolWhenPrintedBecauseItReadsLikeARealSentence` or similar. When on the matter, you may also consider using the [oxford comma](https://annhandley.com/oxford-comma/) 😄

Reply:
I dunno, I am not a native speaker. But when I tested it, the message read like... well bad. It seemed worth to do this.
But then again, the oxford comma article confuses me rather than making my understanding of English better (and people don't believe me, when I share with them what I had read: that English is one of the hardest languages out there). 😄

Reply:
Just suggestions, feel free to ignore them if you find them inconvenient for your case 🆒

Reply:
I put the logic into a separate function. Hope it helps clarity.
And comments are always welcome, to make the thing better. 👍